ajax/libs/multi.js/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "multi.js",
+  "filename": "multi.min.js",
+  "version": "0.2.3",
+  "description": "A user-friendly replacement for select boxes with multiple attribute enabled",
+  "keywords": [
+    "user-friendly",
+    "select",
+    "multiple",
+    "simple"
+  ],
+  "author": {
+    "name": "Fabian Lindfors",
+    "url": "https://fabianlindfors.se"
+  },
+  "homepage": "https://fabianlindfors.se/multijs/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Fabianlindfors/multi.js.git"
+  },
+  "npmName": "multi.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
```

Comment:
We still can use the files under `src`(I test these in local, and everything looks good.)
Would you please also add the following pattern ti grab the files? 
Also, `multi.css` & `multi.js` need to be added, would you add them from `src`?
```js
    {
      "basePath": "src",
      "files": [
        "multi*"
      ]
    }
```

Many thanks.